src/main/java/data/Account.java

```diff
@@ -153,11 +153,27 @@ public void setAggregationMode(String aggregationMode) {
     this.aggregationMode = aggregationMode;
   }
 
+  public static List<String> getAccountSheetHeader() {
+    String[] accountData = { 
+      FormIdNames.ACCOUNT_ID, 
```

Comment:
should this return an ArrayList instead? I can't remember what I read about this in a comment

Reply:
No, returning a List is fine.